solutions_from_participants/lab4/lab4-analysis_Lowe.ipynb

```diff
@@ -0,0 +1,971 @@
+      " -9.08531803e-03  3.46616855e-02 -2.16504323e-02  5.02887915e-03\n",
+      "  1.33609536e-01  5.30798004e-02  6.85355614e-02  1.05791261e-01\n",
+      "  1.87550683e-01 -1.55408740e-03 -6.00949646e-11  1.95028115e-02\n",
+      " -2.25056138e-01  2.78627598e-01]\n",
+      "Expectation values for pub 9: [ 0.43409768  0.01270225  0.01904894  0.02976117  0.05483579  0.2358548\n",
+      "  0.07315758  0.11883017  0.17312561  0.19313185  0.20067673  0.18721224\n",
+      "  0.20454689  0.06456691  0.03700694 -0.00269324 -0.00993189  0.09017418\n",
+      "  0.09698546  0.09502621  0.00183771  0.00928075  0.03671777 -0.00225219\n",
+      "  0.02371339  0.02999577  0.00753556  0.06412855  0.20834158  0.09586917\n",
+      "  0.30116776  0.3296292   0.14967638  0.31080033  0.12303067  0.2059081\n",
+      "  0.11246393  0.00912594  0.00381899  0.00415078  0.0903131   0.04109809\n",
+      "  0.11913365  0.25530225  0.23492658  0.21872152  0.0720249   0.04340217\n",
+      " -0.1132719   0.42943067]\n",
+      "Expectation values for pub 10: [0.71325903 0.32871469 0.48871994 0.2647161  0.22513349 0.77952901\n",
+      " 0.6910616  0.48347016 0.42510833 0.48017195 0.59496192 0.53061209\n",
+      " 0.56925636 0.41739504 0.21828665 0.46988069 0.25514002 0.13293033\n",
+      " 0.25724876 0.46797782 0.43302269 0.57338642 0.46634665 0.75945245\n",
+      " 0.42852461 1.18334116 1.13802324 0.90661326 0.76710916 1.02381697\n",
+      " 0.7452687  0.75380432 0.18716159 0.19132201 0.23552893 0.19143329\n",
+      " 0.66171487 0.28643306 0.53577577 0.0267596  0.08757409 0.26966249\n",
+      " 0.3921535  0.34149912 0.18071977 0.38285744 0.62309644 0.13015639\n",
+      " 0.66189864 0.79779705]\n",
+      "Expectation values for pub 11: [0.89111982 0.72557921 0.78777358 0.76035696 0.54386474 1.18064528\n",
+      " 0.81279895 1.11888804 0.83459676 1.00654168 0.91007769 1.13099822\n",
+      " 1.14930674 1.33673167 1.38846321 1.26046134 1.05537447 1.22753174\n",
+      " 0.97538996 0.87945715 1.01532685 1.08434259 1.30619497 1.10664762\n",
+      " 1.03702092 0.87660169 1.26090444 1.07041063 1.01022208 0.33033208\n",
+      " 0.63610088 0.49490708 0.60254808 0.03960303 0.07902912 0.63519402\n",
+      " 0.48399441 1.13894445 0.99592351 0.71961002 0.5300121  0.84605501\n",
+      " 0.56938121 0.96201701 0.81907649 0.94406781 0.87813229 1.08457021\n",
+      " 0.95256732 0.95394832]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Code to grab all of the expectation values. Anisotropic case: \n",
+    "# Exercise 7: Parse all the data\n",
+    "\n",
+    "from qiskit_ibm_runtime import QiskitRuntimeService\n",
+    "\n",
+    "service = QiskitRuntimeService(\n",
+    "    channel='ibm_quantum',\n",
+    "    instance='events/qgss/24-8',\n",
+    "    token='636dbe2c0dd075571d6bdb21664ddba3c5a4b4f92c0a6e105a8ad389f8a917677a3d34e5813fffd694d7adc18f881640b7f3a4d0db24bfdd43c41858f6639d29'\n",
```

Comment:
hello @mlowe308 can you please make sure to **remove your API Token** and any **other secret data that should not be share**d from your notebook
also i would recommend to [generate a new token](https://quantum.ibm.com/account) since this one has now been compromised: https://quantum.ibm.com/account